ui/src/utils/adapters.ts

```diff
@@ -156,3 +158,61 @@ export const stringBoolean = StrictSchema<string, boolean>()(
     }
   })
 );
+
+// TODO - are the following constants and functions adapters? Parsers? Should they live in "utils/json.ts"? They look out of place here.
```

Comment:
I think they are ok here. This is a generic adapter-utils module. I'd not create another adapter-utils module just to hold `jsonLiteralParser`, `jsonParser`, and `getJsonFromUrl` but this is, of course, subjective.

Reply:
What are the criteria of choosing where to add these types of functions and types? How are some an "adapter", but not a "parser"? In `adapters/api/parsers`, we have `jsonLdTypeParser`, but here we have `jsonLiteralParser`. We also have `iden3JsonLdTypeParser`, but we also have a `utils/iden3.ts`.

I'm confused by the entire file structure and the amount of similarly named functions, types and constants in all these different places and I can't discern what goes where from a logical pov.

Reply:
Thanks for the call and discussion. All further action will be taken as part of [this issue](https://polygonid.atlassian.net/browse/PID-395?atlOrigin=eyJpIjoiNTI1ZGE3OTIxMmZjNDI0ZjgxNDRiOGMxOWU4MmQ5MTciLCJwIjoiaiJ9), as discussed.

Reply:
Thanks to you! As we commented in the meet the name "adapters" was proposed by @elias-garcia and I think it comes from the [hexagonal architecture](https://en.wikipedia.org/wiki/Hexagonal_architecture_(software)).

To be honest, I never liked it much. I was used to naming the layer that integrates external parties into the core of the app (parsing/serializing data) "persistence", but in the end, it's just a name and has no relevance at all.